site/docs/config.md

```diff
@@ -207,38 +179,10 @@ vg.embed('#horizontal_line', {
 | tension       | Number        | Depending on the interpolation type, sets the tension parameter.  [D3's line interpolation document](https://github.com/mbostock/d3/wiki/SVG-Shapes#line_interpolate) |
 
 #### Example: interpolate with `monotone`
-```js
-{
-  "data": {"url": "data/stocks.csv", "formatType":"csv"},
-  "transform": {"filter": "datum.symbol==='GOOG'"},
-  "mark": "line",
-  "encoding": {
-    "x": {"field": "date", "type": "temporal"},
-    "y": {"field": "price", "type": "quantitative"}
-  },
-  "config":{
-    "mark": {"interpolate": "monotone"}
-  }
-}
-```
-<script>
-vg.embed('#line_interpolate', {
-  mode: 'vega-lite',
-  spec: {
-    "data": {"url": "../data/stocks.csv", "formatType": "csv"},
-    "transform": {"filter": "datum.symbol==='GOOG'"},
-    "mark": "line",
-    "encoding": {
-      "x": {"field": "date", "type": "temporal"},
-      "y": {"field": "price", "type": "quantitative"}
-    },
-    "config": {
-      "mark": {"interpolate":"monotone"}
-    }
-  }
-});
-</script>
-<div id="line_interpolate"></div>
+
+<div id="ex-line_monotone" class="side"></div>
+<script>example("line_monotone", "docs")</script>
```

Comment:
That's not true because a lot of examples are in `examples/` as well, so when you want to change one line - you have to change in multiple places (3 times in the old code).